sqlacodegen/codegen.py

```diff
@@ -19,6 +19,12 @@
 from sqlalchemy.types import Boolean, String
 from sqlalchemy.util import OrderedDict
 
+# Support CIText and CIText[] in PostgreSQL via sqlalchemy-citext
+try:
+    from citext import CIText  # noqa: F401
```

Comment:
Is it not enough to import the module?

Reply:
Should be no different since it's not directly used. I'll change that to only import the module